Source/WebKit/UIProcess/API/glib/WebKitUserContentManager.cpp

```diff
@@ -405,9 +405,9 @@ class ScriptMessageClientGtk final : public WebScriptMessageHandler::Client {
     {
     }
 
-    void didPostMessage(WebPageProxy&, FrameInfoData&&, API::ContentWorld&, JavaScriptEvaluationResult&& jsMessage) override
+    void didPostMessage(WebPageProxy&, FrameInfoData&&, API::ContentWorld&, std::optional<JavaScriptEvaluationResult>&& jsMessage) override
     {
-        Ref serializedScriptValue = API::SerializedScriptValue::createFromWireBytes(jsMessage.wireBytes());
+        Ref serializedScriptValue = API::SerializedScriptValue::createFromWireBytes(jsMessage->wireBytes());
```

Comment:
is this optional guaranteed to be not empty?

Reply:
Kind of because the non-Cocoa implementation of JavaScriptEvaluationResult::extract always returns non-nullopt.  That's very fragile though.  I'll add an early return in case that changes.